src/hist/core.py

```diff
@@ -473,3 +473,47 @@ def plot_pull(
         fig.add_axes(pull_ax)
 
         return fig, ax, pull_ax
+
+    def __getitem__(self, index):
+        """
+            Get histogram item.
+        """
+
+        if isinstance(index, dict):
+            return super().__getitem__(index)
+
+        if not hasattr(index, "__iter__"):
+            index = (index,)
+
+        t: tuple = ()
+        for idx in index:
+            if isinstance(idx, complex):
+                t += (loc(idx.imag, int(idx.real)),)
```

Comment:
Great, though I think maybe we should throw an error if the real part is not exactly an integer? Not sure, since it internally obviously it is an float.

Reply:
Yes, I am not sure how to deal with it, either. As the `2j+2` is passed as `2.0` sometimes.

Reply:
It should be a float all the time. But you can check if `idx.real % 1 == 0` and raise an error if not.